src/Telemetry/AzureDevOpsClient/AzureDevOpsClient.cs

```diff
@@ -22,6 +24,11 @@ public sealed class AzureDevOpsClient : IAzureDevOpsClient
         private readonly string _organization;
         private readonly SemaphoreSlim _parallelism;
 
+        private const int _retryDelayMax = 2500;
```

Comment:
'd suggest adding units of measurement here like `_retryDelayMaxMs`

Reply:
Or just use a TimeSpan.

Reply:
He's doing `Random(min, max)` with these so `TimeSpan` here wouldn't make sense, I guess.

But since these numbers are only used in one place, I would hide them away and have something like:
```
private int GetRandomBackOff() => _random.Next(2000, 2500);
```

But this actually might go away with the use of `ExponentialRetry`

Reply:
we went with the `private int GetRandomBackOff() => _random.Next(2000, 2500);`